README.md

```diff
@@ -9,6 +9,7 @@ Android Signature Pad is an Android library for drawing smooth signatures. It us
  * Bézier implementation for a smoother line
  * Variable point size based on velocity
  * Customizable pen color and size
+ * JPEG and SVG formats support
```

Comment:
It should be "Bitmap and SVG support", as the library returns a Bitmap or a SVG string. Bitmap to JPG/PNG/GIF conversion is up to the developer.